### PR TITLE
Check validity of cache and disk size in mount configs during 'occlum build'

### PR DESCRIPTION
**Pull request by lucassong-mh**

### CODE REVIEW

File: tools/gen_internal_conf/src/main.rs [hunk 1]
```diff
@@ -300,6 +300,52 @@ fn main() {
         );
         assert!(kernel_heap_max_size >= kernel_heap_init_size);
 
+        // Check validity of cache and disk size in mount options
```
Comment (volcano0dr) [inline tools/gen_internal_conf/src/main.rs:303]: This check code can be moved to line 478.

File: tools/gen_internal_conf/src/main.rs [hunk 2]
```diff
@@ -300,6 +300,52 @@ fn main() {
...
+                    println!(
+                        "Invalid cache_size \"{}\". The cache_size must be 1. larger than the minimum size \"{}\", \
+                        2. aligned with cache page size \"{}\", \
+                        3. smaller than the kernel_heap_max_size \"{}\".",
+                        cache_size, MIN_CACHE_SIZE, CACHE_PAGE_SIZE, kernel_heap_max_size
+                    );
+                    return;
+                }
+            }
+            if let Some(disk_size_str) = mount.options.disk_size.as_ref() {
```
Comment (volcano0dr) [inline tools/gen_internal_conf/src/main.rs:330]: Need to check if it is Ext2Fs must set `disk_size`.

File: tools/gen_internal_conf/src/main.rs (hunk 1 above)
Comment (lucassong-mh) [reply tools/gen_internal_conf/src/main.rs]: Done

File: tools/gen_internal_conf/src/main.rs (hunk 2 above)
Comment (lucassong-mh) [reply tools/gen_internal_conf/src/main.rs]: Done

**Review by volcano0dr — APPROVED** [review]
LGTM

**Pull request merged**